lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -12,12 +12,13 @@ def initialize(_raven_configuration)
       @data_sync = GovukDataSync.new(ENV["GOVUK_DATA_SYNC_PERIOD"])
       self.active_sentry_environments = []
       self.data_sync_excluded_exceptions = []
-      self.should_capture = ignore_exceptions_based_on_env_and_data_sync
+      self.before_send = ->(error_or_event, _hint) { error_or_event }
```

Comment:
Hmmm, if I've understood https://github.com/alphagov/govuk_app_config/pull/196#discussion_r629452794 then we could avoid the stubbed lambda by swapping the above with this:

```ruby
      self.before_send = default_before_send_actions
```

...and then the method would be:

```ruby
def before_send=(closure)
  super(closure)
end
```

That does seem deceptively simple though, so I suspect it would be overriding previous lambdas rather than merging them. Perhaps one to consider refactoring after writing the "two `before_send=` test" suggested above.

Reply:
Yeah, because it's overriding then the `before_send=` method needs to do the work of merging if a non-default one is chosen; however, that also applies when the default one is being set and leads to doubling the default — if and only if a non-default one is not set (hence this stub).

What would be easiest (and what I assumed at first) would be if `before_send=` is only defined _after_ the `initialize` is called; or otherwise, if the method could check whether the default is being set. But that ends up needing to do some weird reflection or state (a `before_send_was_overriden` instance variable or something?).